airflow/providers/amazon/aws/operators/redshift_data.py

```diff
@@ -119,6 +120,22 @@ def execute_query(self):
         resp = self.hook.conn.execute_statement(**trim_none_values(kwargs))
         return resp['Id']
 
+    def execute_batch_query(self):
+        if self.sql is None:
+            AirflowException("Missing require param sqls")
```

Comment:
There ia no sqls param any more
Also curious why we need this exception (we dont have one in execute_query)
What happens if empty query is submited to the API?

Reply:
- AWS API will through ```Invalid type for parameter Sqls, value: , type: <class 'str'>, valid types: <class 'list'>, <class 'tuple'>```

- And if a user will not pass `sql` param then baseoperator will catch it ```airflow.exceptions.AirflowException: missing keyword argument 'sql'```

So, yeah you are right this exception will not add value here